User-Help/DeletedPostHelp

```diff
@@ -0,0 +1,15 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| deleted | Site Information |
+
+## Why are posts deleted? 
+
+Posts are deleted for various reasons. They may be deleted either by a moderator or by the original poster.
```

Comment:
Should probably mention community deletions here -- "...by community curators or by the original poster" would cover all cases I think.  I'm sorry I didn't notice this before.